homeassistant/components/flux_led/strings.json

```diff
@@ -27,7 +27,7 @@
         "data": {
           "mode": "The chosen brightness mode.",
           "custom_effect_colors": "Custom Effect: List of 1 to 16 [R,G,B] colors. Example: [255,0,255],[60,128,0]",
-          "custom_effect_speed_pct": "Custom Effect: Speed in percents for the effect that switch colors.",
+          "custom_effect_speed_pct": "Custom Effect: Speed in percentage for the effect that switch colors.",
```

Comment:
Might need further rewording for grammar:
E.g.
'Speed in percentage for the effects that switch color'
Or
'Speed in percentage for the effect that switches color'
Or
'Color effect switching speed (%)'

Or similar

Reply:
Done, thanks!